### DISCUSSION TITLE
react-spring document

### DISCUSSION THREAD

**Discussion by tarkers** [pmndrs/react-spring#1500] [Support]
## 🤓 Question
folllow react-spring document
```js
function Number() {
  const [flip, set] = useState(false)
  const { number } = useSpring({
    reset: true,
    reverse: flip,
    from: { number: 0 },
    number: 1,
    delay: 200,
    config: config.molasses,
    onRest: () => set(!flip),
  })

  return <animated.div>{number.to(n => n.toFixed(2))}</animated.div>
}
```
## Error
```
TypeError: number.to is not a function
```
should the document need update or i lost to follow sth?

### use
"react": "^17.0.2",
"react-spring": "^8.0.27"
(You _must_ search the issues before asking your question. Please consider asking in [the Discussions tab](https://github.com/pmndrs/react-spring/discussions) and/or [Stack Overflow](https://stackoverflow.com) first.)

**Comment by joshuaellis** [comment]
You're using v8, try `number.interpolate`
